File: Data Structures & Algorithms/last-stone-weight/submission-0.java
class Solution {
    public int lastStoneWeight(int[] stones) {
        PriorityQueue<Integer> maxHeap = new PriorityQueue<>( (a, b)-> b - a );
        for (int i : stones) {
            maxHeap.add(i);
        }
        while (maxHeap.size() > 1) {
            Integer stoneA = maxHeap.poll();
            Integer stoneB = maxHeap.poll();
            if (stoneA != stoneB) {
                Integer newStone = Math.abs(stoneB - stoneA);
                maxHeap.add(newStone);
            }
        }

        if (maxHeap.size() == 1) return maxHeap.poll();

        return 0;
    }
}

File: Data Structures & Algorithms/score-of-a-string/submission-0.cs
public class Solution {
    public int ScoreOfString(string s) {
        var score = 0;
        for (int i = 1; i < s.Length; i++) {
            score += Math.Abs((int)s[i] - (int)s[i-1]);
        }
        return score;
    }
}